zaas-client/src/test/java/org/zowe/apiml/zaasclient/service/internal/ZaasClientImplHttpsTests.java

```diff
@@ -347,6 +347,19 @@ void testPassTicketWithToken_ValidToken_ValidPassTicket() throws Exception {
         assertEquals("ticket", passTicketService.passTicket(token, "ZOWEAPPL"));
     }
 
+    @Test
+    void testPassTicketWithToken_InValidApplicationID_throwsException() throws Exception {
+
+        ZaasPassTicketResponse zaasPassTicketResponse = new ZaasPassTicketResponse();
+
+        when(zaasHttpsClientProvider.getHttpClient()).thenReturn(closeableHttpClient);
+        when(httpsEntity.getContent()).thenReturn(new ByteArrayInputStream(new ObjectMapper().writeValueAsBytes(zaasPassTicketResponse)));
+        when(closeableHttpResponse.getStatusLine().getStatusCode()).thenReturn(500);
+
+        ZaasClientException exception = assertThrows(ZaasClientException.class, () -> passTicketService.passTicket(token, "XBADAPPL"));
```

Comment:
It look like this test checks passTicketService. Could you create another test, that call zaasClient and verify its response?